MapboxDirections/OfflineDirections.swift

```diff
@@ -84,24 +90,23 @@ extension Directions: OfflineDirectionsProtocol {
     }
     
     /**
-     Initializes an `URLSessionDownloadTask` used for downloading tiles within a given bounding box.
+     Downloads offline routing tiles of the given version within the given coordinate bounds, using the shared URLSession.
      
      - parameter coordinateBounds: The bounding box
      - parameter version: The version to download. Version is represented as a String (yyyy-MM-dd-x)
-     - parameter session: Optional URLSession
      - parameter completionHandler: The closure to call with the results
-     
-     - returns: A `URLSessionDownloadTask`
      */
-    @discardableResult
-    @objc public func downloadTiles(in coordinateBounds: CoordinateBounds,
+    public func downloadTiles(in coordinateBounds: CoordinateBounds,
                               version: OfflineVersion,
-                              session: URLSession? = URLSession.shared,
-                              completionHandler: @escaping OfflineDownloaderCompletionHandler) -> URLSessionDownloadTask {
-        
-        let urlSession = session ?? URLSession.shared
+                              completionHandler: @escaping OfflineDownloaderCompletionHandler) -> () {
+        let task = routingTilesDataTaskWith(coordinateBounds, version: version, completionHandler: completionHandler)
+        return task.resume()
+    }
+
+    @objc(routingTilesDataTaskWithCoordinateBounds:version:completion:)
```

Comment:
If we make this public, let’s add documentation so it shows up in jazzy.

Reply:
👍

Reply:
do you agree that it's worth it to expose this publicly? I wanted to address #352 but also preserve finer-grained control for developers

Reply:
Sure, that’s fine.